tests/command_line/test_bader_caller.py

```diff
@@ -127,7 +127,7 @@ def test_atom_parsing(self):
         assert len(analysis.atomic_densities) == len(analysis.chgcar.structure)
 
         assert np.sum(analysis.chgcar.data["total"]) == approx(
-            np.sum([dct["data"] for dct in analysis.atomic_densities])
+            np.sum([np.sum(dct["data"]) for dct in analysis.atomic_densities])
```

Comment:
I misunderstood the implementation a little. The implementation seems to calculate the total num instead of element-wise sum:
https://github.com/materialsproject/pymatgen/blob/dc60d35a21fd7b55d82c65bb078afc4a2c0e9bbf/pymatgen/command_line/bader_caller.py#L209-L216

The `np.sum` must have been removed by accident in ff45527caf5de3bc76ed859bab0d978cab38972d, adding it back is what we need to fix this.